source/Calamari.AzureAppService/DeployAzureAppServiceCommand.cs

```diff
@@ -23,7 +22,7 @@ public class DeployAzureAppSettingsCommand : PipelineCommand
     {
         protected override IEnumerable<IDeployBehaviour> Deploy(DeployResolver resolver)
         {
-            yield return resolver.Create<AzureAppServiceBehaviour>();
+            yield return resolver.Create<AzureAppServiceDeployBehaviourFactory>();
```

Comment:
This seems _wrong_ for the `DeployAzureAppSettingsCommand` 🤔 
But I didn't want to stuff around with it, so I just kept it calling the same behaviours